src/main/java/io/socket/engineio/client/Transport.java

```diff
@@ -148,6 +151,7 @@ protected void onClose() {
 
     public static class Options {
 
+        public storeOkHttpClient storeOkHttpClient = null;
```

Comment:
I think that use [WebSocket.Factory](http://square.github.io/okhttp/3.x/okhttp/okhttp3/WebSocket.Factory.html) is enough.